src/js/integrations/nativelinkederrors.ts

```diff
@@ -227,81 +229,3 @@ export class NativeLinkedErrors implements Integration {
     return NATIVE.fetchNativeStackFramesBy(instructionsAddr);
   }
 }
-
-// TODO: Needs to be exported from @sentry/browser
```

Comment:
Since this is exported now, did we need to adapt the code or is it the very same code? otherwise, the adaptations have to remain here.

Reply:
It was copied without a change. So export from JS will work.